pkg/engine/response/response.go

```diff
@@ -20,6 +25,9 @@ type EngineResponse struct {
 
 	// Policy Response
 	PolicyResponse PolicyResponse
+
+	// NamespaceLabels given by policy context
+	NamespaceLabels map[string]string
```

Comment:
Why add `NamespaceLabels` to `EngineResponse`? Can we get it from `EngineResponse.Policy`?

Reply:
So `EngineResponse.NamespaceLabels` are the labels for the namespaces of the resources to be validated. It is passed from `PolicyContext.NamespaceLabels`,  see [here](https://github.com/Eileen-Yu/kyverno/blob/b376d55aea/pkg/engine/validation.go#L45).

Which should be different from the one of `EnginResponse.Policy`?

Reply:
Make sense.